tools/runners/test262-harness.py

```diff
@@ -861,15 +862,15 @@ def run(self, command_template, tests, print_summary, full_summary, logname, job
             if job_count == 0:
                 job_count = None # uses multiprocessing.cpu_count()
 
-            pool = multiprocessing.Pool(processes=job_count, initializer=pool_init)
-            try:
-                for result in pool.imap(test_case_run_process, cases):
-                    if logname:
-                        self.write_log(result)
-                    progress.has_run(result)
-            except KeyboardInterrupt:
-                pool.terminate()
-                pool.join()
+            with multiprocessing.Pool(processes=job_count, initializer=pool_init) as pool:
+                try:
+                    for result in pool.imap(test_case_run_process, cases):
+                        if logname:
+                            self.write_log(result)
+                        progress.has_run(result)
+                except KeyboardInterrupt:
+                    pool.terminate()
+                    pool.join()
```

Comment:
is the try-except construct still needed? when multiprocessing Pool is used as a context manager, it automatically calls terminate on exit (even for other types of exceptions, not only for KeyboardInterrupts)

Reply:
no, I do not think its still needed